### ISSUE TITLE
Feature Request: Lower resolution

### ISSUE THREAD

**Issue by relishcolouredhat** [opsxcq/meme-vibing-cat#1]
Fantastic repo, but sometimes I need to shitpost **faster**. 

Can we support a lower resolution option?

**Comment by opsxcq** [comment]
Yes, also there is a pending contribution to reduce the video duration, it will be merged this weekend, so smaller file size = faster upload. Also reducing the resolution can be applied. Soon you will be able shit post at light speed !
